src/main/resources/reference.conf

```diff
@@ -122,6 +122,10 @@ zombieClusterMonitor {
   creationHangTolerance = 1 hour
 }
 
+clusterServiceMonitor {
```

Comment:
Minor: maybe name this `clusterToolMonitor` to match the class name it corresponds with